stable/cert-manager/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: cert-manager
-version: v0.3.1
+version: v0.3.2
```

Comment:
As this change adds a feature, I'd recommend bumping the minor version.

Reply:
I believe we follow the guidelines from https://semver.org/, which indicate: ```Minor version Y (x.Y.z | x > 0) MUST be incremented if new, backwards compatible functionality is introduced to the public API.``` (https://semver.org/#spec-item-7)

I understand your reasoning regarding app version, but I think that a specific version of a chart is meant for a specific version of an application, as noted with the `appVersion` key in the Chart.yaml. As far as I know, most other charts do not tie their Chart version to specific app version ranges, but if this is important to you, it may make more sense to simply have a table in the README instead, as that's the first documentation a user gets to see. I personally appreciate the table provided by the cluster-autoscaler (https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler#releases).

Reply:
I've been pushing for following the semver guidelines in the last months without much back push, so I suspect it's more or less accepted ;-) That said, having a proper guideline about it would indeed be best, one way or the other.

Also, I think that `appVersion` is actually a string, so you might get away with simply having a range of versions in there?

Regarding my example, yes, it's subtly different, but the same solution could be used for this problem as well. I don't think updating the table in the README is that much more tedious than updating any other part of the documentation after a change.

I'm not necessarily against it, mind you, I just prefer to follow a clear and defined standard in this regard. I'm hoping that the helm client will get support for warnings based on semver versions, so a user can make a more informed decision with regards to updates. But I have no idea if such a mechanism is even considered at this time.

Reply:
Hey, any updates here?

I'm happy to follow the upstream guidance, however I'd like to see that guidance make its way to the 'helm best practices' website beforehand, as I am reluctant to break/change our current versioning pattern unless it is absolutely decided on how versioning should be handled, as this will be an irreversible change which does not 'solve' any of the issues users have complained to me about regarding how we version our chart 😄.

Is there some other issue/thread where this is being discussed? As it stands, I cannot push any updates to our Helm chart until this issue is resolved 😄